.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+/appinfo/info.xml @max-nextcloud @vitormattos
+* @max-nextcloud
```

Isn't necessary, only to be best to look
```suggestion
/appinfo/info.xml @max-nextcloud @vitormattos
*                 @max-nextcloud
```